### DISCUSSION TITLE
Error make build-all

### DISCUSSION THREAD

**Discussion by MrMorlarty** [community/community#119826] [Programming Help]
### Select Topic Area

Bug

### Body

Greetings! An error occurs when building Docker. When trying to build make build-all, 2 errors appear. We were told that it was worth trying through make proto-gen. But errors also appear and the assembly does not work. We were also told that if proto-gen does not work, then on the release page on GH there is a build of definitions for protobuf protobuf-prebuild.tar.gz But we don’t know what to do with it. Help me please.
!

[IMG_1351](https://github.com/community/community/assets/167234068/02f0fbae-58f6-4009-abad-eddc1e4d22b7)
![IMG_1352](https://github.com/community/community/assets/167234068/5d5924f0-1edc-44ee-b8bf-1f5db5dd9f61)
![IMG_1353](https://github.com/community/community/assets/167234068/46b469c5-5b8c-4565-944d-a4c57e98ffbc)

**Comment by ebndev** [comment]
Thanks for posting in the GitHub Community, @MrMorlarty! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Best of luck!
